### PR TITLE
docs: Create usage.md

### PR DESCRIPTION
**Pull request by Ccode-lang**

### CODE REVIEW

File: usage.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+```
+or  
+```bash
+# if v is not on path
+<path to v> <name of file2>.v
+```
+  
+  
+# More options
+puting file2 as ```-``` will output the v code to the terminal.
```
Comment (div72) [inline usage.md:18]:
```suggestion
 Using `-` as the output argument will output the transpiled code to stdout.
```

**Review by div72 — CHANGES_REQUESTED** [review]
Instructions for compiling the V output is unnecesarry and out of scope for py2v. A user should already have an idea on how to use V before using py2v.

**Comment by div72** [comment]
@Ccode-lang Please also remove the instructions for compiling the code.

**Comment by Ccode-lang** [comment]
ok.

**Pull request merged**